src/components/ToggleSection.tsx

```diff
@@ -39,20 +39,20 @@ const ToggleSection = ({ title, onClick = () => null, children, className }: Tog
 
   return (
     <div className={className}>
-      <CellControl>
+      <Title>
         {title}
         <Toggle onToggle={handleToggle} label={title} toggled={isShown} />
-      </CellControl>
-      <CellChildren
+      </Title>
+      <Content
         animate={{
           height: isShown ? 'auto' : 0,
           opacity: isShown ? 1 : 0,
           visibility: isShown ? 'visible' : 'hidden'
         }}
         transition={{ duration: 0.2 }}
       >
-        <CellChildrenInner>{children}</CellChildrenInner>
-      </CellChildren>
+        <Children>{children}</Children>
+      </Content>
     </div>
```

Comment:
Unnecessary renames

Reply:
Why so cold ? :/ 

The component names are shorter, so why not.

NIT: If it was me I'd just rename "Content" into "SectionContent" for extra clarity, as a mention to the ToogleSection name.